src/_repobee/cli/preparser.py

```diff
@@ -46,11 +47,18 @@ def parse_args(
         prog="repobee", description="plugin pre-parser for _repobee."
     )
 
+    add_arguments(parser)
+    args = parser.parse_args(sys_args)
+
+    return args
+
+
+def add_arguments(parser: argparse.ArgumentParser,) -> None:
```

Comment:
This function must take `default_config_file` as an argument, like `preparser.parse_args()` does. As it stands right now, `preparser.parse_args()` does not respect the config file argument.

In addition, this is a public function of this module, could you document it according to [these instructions](https://docs.repobee.org/en/stable/contributing.html#code-style)?

Reply:
🤦 good point. done.